numpy/core/src/umath/ufunc_object.c

```diff
@@ -4310,9 +4306,9 @@ PyUFunc_GenericReduction(PyUFuncObject *ufunc,
                         "reduceat does not allow multiple axes");
             goto fail;
         }
-        ret = (PyArrayObject *)PyUFunc_Reduceat(ufunc, mp, indices, out,
-                                            axes[0], otype->type_num);
-        Py_DECREF(indices);
+        ret = (PyArrayObject *)PyUFunc_Reduceat(ufunc,
+                mp, indices, out, axes[0], otype->type_num);
```

Comment:
You could also put `ufunc, ` on this line.

Reply:
Yeah, I have been using this style quite a bit for `self`-like arguments (keeping them on the same line).

Happy to try and change that habit, or you may just have to do a style cleanup at some point :).